ver/jp/splat.yaml

```diff
@@ -34,199 +36,3197 @@ segments:
+    - [0x431E0, c, os/motor, gcc_272 -O3]
+    - [0x43570, c, os/pfsallocatefile, gcc_272 -O3]
+    - [0x439A0, c, os/pfsdeletefile, gcc_272 -O3]
+    - [0x43BC0, c, os/pfsreadwritefile, gcc_272 -O3]
+    - [0x44000, c, os/pfsfilestate, gcc_272 -O3]
+    - [0x441A0, c, os/pfsfindfile, gcc_272 -O3]
+    - [0x44360, c, os/pfsfreeblocks, gcc_272 -O3]
+    - [0x44470, c, os/pfsinitpak, gcc_272 -O3]
+    - [0x44840, c, os/pfsrepairid, gcc_272 -O3]
+    - [0x448A0, c, os/pfsgetstatus, gcc_272 -O3]
+    - [0x44AB0, c, os/pfsselectbank, gcc_272 -O3]
+    - [0x44B20, c, os/contpfs, gcc_272 -O3]
+    - [0x455B0, c, os/contramread, gcc_272 -O3]
+    - [0x457A0, c, os/contramwrite, gcc_272 -O3]
+    - [0x459B0, hasm, os/parameters]
+    - [0x45A10, c, os/afterprenmi, gcc_272 -O3]
+    - [0x45A30, c, os/initialize, gcc_272 -O3]
+    - [0x45DD0, hasm, os/exceptasm]
+    - [0x46740, hasm, os/__osDisableInt]
+    - [0x467B0, hasm, os/__osRestoreInt]
+    - [0x467D0, c, os/setglobalintmask, gcc_272 -O3]
+    - [0x46810, c, os/resetglobalintmask, gcc_272 -O3]
+    - [0x46860, c, os/pirawdma, gcc_272 -O3]
+    - [0x46930, c, os/pigetcmdq, gcc_272 -O3]
+    - [0x46950, c, os/epirawread, gcc_272 -O3]
+    - [0x46AC0, c, os/epirawwrite, gcc_272 -O3]
+    - [0x46C30, c, os/ai, gcc_272 -O3]
+    - [0x46C50, hasm, os/bcmp]
+    - [0x46D60, c, os/xlitob, gcc_272 -O3]
+    - [0x46FB0, c, os/xldtob, gcc_272 -O3]
+    - [0x47A40, hasm, os/__osGetCause]
+    - [0x47A50, hasm, os/__osGetSR]
+    - [0x47A60, hasm, os/__osSetCompare]
+    - [0x47A70, hasm, os/setfpccsr]
+    - [0x47A80, hasm, os/setsr]
+    - [0x47A90, hasm, os/__osSetWatchLo]
+    - [0x47AA0, c, os/sp, gcc_272 -O3]
+    - [0x47AC0, c, os/spgetstat, gcc_272 -O3]
+    - [0x47AD0, c, os/spsetstat, gcc_272 -O3]
+    - [0x47AE0, c, os/spsetpc, gcc_272 -O3]
+    - [0x47B10, c, os/sprawdma, gcc_272 -O3]
+    - [0x47BA0, c, os/sirawread, gcc_272 -O3]
+    - [0x47BF0, c, os/sirawwrite, gcc_272 -O3]
+    - [0x47C40, c, os/crc, gcc_272 -O3]
+    - [0x47D30, c, os/destroythread, gcc_272 -O3]
+    - [0x47E10, hasm, os/osMapTLBRdb]
+    - [0x47E70, c, os/vi, gcc_272 -O3]
+    - [0x47F80, c, os/vigetcurrcontext, gcc_272 -O3]
+    - [0x47F90, hasm, os/padding]
+    - [0x47FA0, hasm, os/guNormalize]
+    - [0x48000, c, os/pfschecker, gcc_272 -O3]
+    - [0x48720, c, os/pfsisplug, gcc_272 -O3]
+    - [0x48A00, c, os/ldiv, gcc_272 -O3]
+    - [0x48BC0, c, os/osSiDeviceBusy]
+    - [0x48BE0, c, gcc/divdi3, -g2]
+    - [0x491A0, c, gcc/moddi3, -g2]
+    - [0x496F0, c, gcc/udivdi3, -g2]
+    - [0x49C50, c, gcc/umoddi3, -g2]
+    - [0x4A120, c, os/strcpy, gcc_272 -O1]
+    - [0x4A150, c, os/strcmp, gcc_272 -O1]
+    - [0x4A190, c, os/memset, gcc_272 -O1]
+    - [0x4A1D0, hasm, os/decode_yay0]
+    - [0x4A280, c, os/osFlash]
+    - [0x4AC70, pm_effect_loads, effect_loads]
+    - [0x4DF10, c, battle/battle]
+    - [0x4E580, bin, n_aspMain_text]
+
+    - [0x4F1E0, .data, main_pre] # TODO: find data offset
```

Comment:
you can just make these `auto` instead imo. but if you do wanna find the offset, go for it